src/FileStorage.php

```diff
@@ -11,16 +11,16 @@
 
 namespace Translation\SymfonyStorage;
 
-use Symfony\Bundle\FrameworkBundle\Translation\TranslationLoader as SymfonyTranslationLoader;
 use Symfony\Component\Translation\MessageCatalogue;
 use Symfony\Component\Translation\MessageCatalogueInterface;
+use Symfony\Component\Translation\Reader\TranslationReader;
```

Comment:
This class does not exist in sf 3.x.

Reply:
@Nyholm that's an interesting case :) TranslationReader does exist since 3.4, but TranslationLoader is deprecated since 3.4 and exists only until 4.0. So, with TranslationReader we can continue supporting the latest version of 3.x and the new versions of 4.x. But if you also want to still maintain 2.8, 3.0, 3.1, 3.2, and 3.3 (I see in Travis config you do it now) - we need to write more legacy code to support those versions. So to move forward, I need to know what do you have in mind about it and what is your further strategy for maintenance of this bundle?

Reply:
A good practice is to support the last 2 LTS versions of symfony. I think we need to write more code here to support both sf 4.0 and sf 2.8.

Reply:
Or alternatively You create 0.5 and say if one wants to use this on sf older than 3.4 it should use 0.3.2. There is not a lot of commits here so this could work.

Reply:
Yeah, That is an option. But Im not sure I want to go down that route. Better to try to support all versions in master.